tekton/release.sh

```diff
@@ -114,7 +114,7 @@ git push --force ${PUSH_REMOTE} release-${RELEASE_VERSION}
 kubectl create namespace ${TARGET_NAMESPACE} 2>/dev/null || true
 
 for task in ${CATALOG_TASKS};do
-    kubectl -n ${TARGET_NAMESPACE} apply -f https://raw.githubusercontent.com/tektoncd/catalog/master/task/golang-${task}/0.1/golang-${task}.yaml
+    kubectl -n ${TARGET_NAMESPACE} apply -f https://raw.githubusercontent.com/tektoncd/catalog/master/task/golangci-${task}/0.1/golangci-${task}.yaml
```

Comment:
for build and test, it is golang-*
for lint it is golangci-*

We need to handle this properly :D

Reply:
since we removed lint i think we can actually remove the install of golangci

Reply:
i see #1025 but it is still not merged

Reply:
@pradeepitm12 If we get #1025 in, then we don't need this PR, all good

Reply:
oh that's nice, let's get @1025 in